README.md

```diff
@@ -24,7 +24,7 @@ Another use is to align blocks of code by `=` or `=>`, etc.
 
 ```sh
 $ go get github.com/Guitarbum722/align
-$ go install
+$ make install
```

Comment:
I'd likely add a line in the README that shows how to build a release with `make release` since that will build all binaries.